views/js/runner/plugins/navigation/next/nextItemWarning.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+/**
+ * Test Runner Navigation Plugin : nextItemWarning
+ *
+ * @author Martin Nicholson <martin@taotesting.com>
+ */
+define([
+    'jquery',
+    'lodash',
+    'i18n',
+    'taoTests/runner/plugin',
+    'taoQtiTest/runner/helpers/currentItem',
+    'taoQtiTest/runner/plugins/navigation/next/dialogConfirmNext'
+], function ($, _, __, pluginFactory, currentItemHelper, dialogConfirmNext){
+    'use strict';
+
+    /**
+     * Returns the configured plugin
+     */
+    return pluginFactory({
+        name : 'nextItemWarning',
```

Comment:
I'm thinking if we shouldn't rename it to `linearNextItemWarning` so people doesn't get mistaken (with the other plugins that handle warnings)

Reply:
I've gone ahead with the name change to `linearNextItemWarning` in all appropriate places.